components/restapi/core/accounts.go

```diff
@@ -27,7 +27,7 @@ func congestionByAccountAddress(c echo.Context) (*api.CongestionResponse, error)
 		workScores = append(workScores, workScore)
 	}
 
-	commitment, err := deps.RequestHandler.GetCommitmentByID(commitmentID)
+	commitment, err := deps.RequestHandler.GetCommitmentByID(queryCommitmentID)
```

Comment:
`queryCommitment` then? :grimacing:

Reply:
but that's `commitmentID`

Reply:
I meant the returned commitment:
```suggestion
	queryCommitment, err := deps.RequestHandler.GetCommitmentByID(queryCommitmentID)
```